alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2702,7 +2708,8 @@ public static function handleLiveReportExportFinished(BatchJob $dbBatchJob, kLiv
 		// Create email params
 		$time = date("m-d-y H:i", $data->timeReference + $data->timeZoneOffset); 
 		$email_id = MailType::MAIL_TYPE_LIVE_REPORT_EXPORT_SUCCESS;
-		$params = array($dbBatchJob->getPartner()->getName(), $time, $dbBatchJob->getId(), $url);
+		$validUntil = date("m-d-y H:i", $data->timeReference + $expiry + $data->timeZoneOffset);
+		$params = array($dbBatchJob->getPartner()->getName(), $time, $dbBatchJob->getId(), $url, self::LIVE_REPORT_EXPIRY_TIME_IN_DAYS, $validUntil);
```

Comment:
I refactored it a bit on [PR#2327](https://github.com/kaltura/server/pull/2327) so kConf is being used.